checker/mta_sts.go

```diff
@@ -132,21 +132,25 @@ func validateMTASTSPolicyFile(body string, result *Result) map[string]string {
 	policy := getKeyValuePairs(body, "\n", ":")
 
 	if policy["version"] != "STSv1" {
-		result.Failure("Policy version must be STSv1")
+		result.Failure("Your MTA-STS policy file version must be STSv1.")
 	}
 
 	if policy["mode"] == "" {
-		result.Failure("Policy file must specify mode")
+		result.Failure("Your MTA-STS policy file must specify mode.")
 	}
-	if m := policy["mode"]; m != "enforce" && m != "testing" && m != "none" {
-		result.Failure("Mode must be one of 'enforce', 'testing', or 'none', got %s", m)
+	if m := policy["mode"]; m == "testing" {
+		result.Warning("You're still in \"testing\" mode; senders won't enforce TLS when connecting to your mailservers. We recommend switching from \"testing\" to \"enforce\" to get the full security benefits of MTA-STS, as long as it hasn't been affecting your deliverability.")
+	} else if m == "none" {
+		result.Warning("@sydneyli message for \"none\" mode here")
```

Comment:
```suggestion
		result.Failure("MTA-STS policy is in \"none\" mode; senders won't enforce TLS when connecting to your mailservers.")
```

Something like this? Changing it to Failure because having this in `none` mode generally suggests that the implementer is migrating off of MTA-STS.